docker/entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+export PGPASSWORD="$POSTGRES_PASSWORD"
+
+until psql -h "$POSTGRES_HOST" -U "$POSTGRES_USERNAME" -d "$POSTGRES_DATABASE" -c '\q'; do
+  >&2 echo "Waiting for Postgres to be ready..."
+  sleep 1
+done
+
+if [ ! -f /automatisch/storage/.env ]; then
+  >&2 echo "Saving environment variables"
+  ENCRYPTION_KEY="${ENCRYPTION_KEY:-$(openssl rand -base64 36)}"
+  APP_SECRET_KEY="${APP_SECRET_KEY:-$(openssl rand -base64 36)}"
+  echo "$ENCRYPTION_KEY
```

Comment:
Here, I have realized that we are not creating `key=value` pairs. It currently looks as follows without explicitly passing corresponding environment variables;

```
automatisch_encryption_key
  automatisch_app_secret_key
``` 

I expect that they should look like the following;

```
ENCRYPTION_KEY=automatisch_encryption_key
APP_SECRET_KEY=automatisch_app_secret_key
```

Reply:
I am resolving this comment as I cannot reproduce what I stated above. The `.env` file does look fine in the sense of structure.